### PR TITLE
Feature/guess location score button animation

### PR DESCRIPTION
**Pull request by amattei777**

### CODE REVIEW

File: app/src/main/java/com/github/wnder/GuessLocationActivity.java [hunk 1]
```diff
@@ -351,7 +357,7 @@ private void updateCompassMode(){
     }
 
     private void compassModeSetup(){
-        TextView compassModeButtonView = (TextView) findViewById(R.id.compassMode);
+        View compassModeButtonView = findViewById(R.id.compassMode);
```
Comment (rkuenzi-epfl) [inline app/src/main/java/com/github/wnder/GuessLocationActivity.java:360]: There is a FloatingActionButton that you should probably use.

File: app/src/main/java/com/github/wnder/GuessLocationActivity.java [hunk 2]
```diff
@@ -370,7 +376,7 @@ private void compassModeSetup(){
             sensorManager.registerListener(listener, (Sensor) list.get(0), SensorManager.SENSOR_DELAY_NORMAL);
             updateGuessPositionFromGPS.run();
             mapboxMap.getUiSettings().setRotateGesturesEnabled(false);
-            compassModeButtonView.setText(R.string.switchCompassModeText);
+            compassModeButtonView.setForeground(getDrawable(R.drawable.ic_outline_explore_24));
```
Comment (rkuenzi-epfl) [inline app/src/main/java/com/github/wnder/GuessLocationActivity.java:379]: Using FloatingActionButton you can the use setImageResource that should do what you want here.

File: app/src/main/java/com/github/wnder/GuessLocationActivity.java [hunk 3]
```diff
@@ -392,7 +398,7 @@ private void switchMode() {
             updateCompassMode();
             sensorManager.unregisterListener(listener);
             mapboxMap.getUiSettings().setRotateGesturesEnabled(true);
-            compassModeButtonView.setText(R.string.switchNormalModeText);
+            compassModeButtonView.setForeground(getDrawable(R.drawable.ic_outline_explore_off_24));
```
Comment (rkuenzi-epfl) [inline app/src/main/java/com/github/wnder/GuessLocationActivity.java:401]: Same as above

File: app/src/main/java/com/github/wnder/GuessLocationActivity.java [hunk 4]
```diff
@@ -410,8 +416,8 @@ private void confirmButton() {
             guessConfirmed = true;
 
             findViewById(R.id.compassMode).setVisibility(View.INVISIBLE);
-            TextView confirmButtonView = (TextView) findViewById(R.id.confirmButton);
-            confirmButtonView.setText(R.string.confirmButtonPressedOnce);
+            View confirmButtonView = findViewById(R.id.confirmButton);
+            confirmButtonView.setForeground(getDrawable(R.drawable.ic_baseline_military_tech_24));
```
Comment (rkuenzi-epfl) [inline app/src/main/java/com/github/wnder/GuessLocationActivity.java:420]: Same as above

**Review by rkuenzi-epfl — CHANGES_REQUESTED** [review]
Please try what I propose, if it doesn't work then we'll keep what you have now if it displays correctly.

**Review by BoostedBoat — CHANGES_REQUESTED** [review]
Once you click on guess, you can't see the image anymore + once you've clicked on guess you can't skip/go to the next picture to guess anymore (you have to go back to the navigation activity). That's not very smooth-friendly I guess

**Comment by BoostedBoat** [comment]
Otherwise the code seems nice to me

File: app/src/main/java/com/github/wnder/GuessLocationActivity.java (hunk 1 above)
Comment (amattei777) [reply app/src/main/java/com/github/wnder/GuessLocationActivity.java]: done

File: app/src/main/java/com/github/wnder/GuessLocationActivity.java (hunk 2 above)
Comment (amattei777) [reply app/src/main/java/com/github/wnder/GuessLocationActivity.java:381]: Well I tried to do that but I couldn't change the size of the icon properly and change it in white color with that function. For now I would like to let it that way. And maybe someone will have to double check the whole app UI latter to homogenize better.

Comment (rkuenzi-epfl) [reply app/src/main/java/com/github/wnder/GuessLocationActivity.java:381]: I think it's fine then, the important thing here is the look I guess.

**Review by rkuenzi-epfl — APPROVED** [review]

**Comment by amattei777** [comment]
> 
> 
> Once you click on guess, you can't see the image anymore + once you've clicked on guess you can't skip/go to the next picture to guess anymore (you have to go back to the navigation activity). That's not very smooth-friendly I guess

I agree that for the user we should add a miniature of the image on that activity somewhere. However we did discuss that point at the neginning of the week and we decide to not to it this week

**Review by BoostedBoat — APPROVED** [review]

**Pull request merged**